content/en/docs/concepts/architecture/controller.md

```diff
@@ -113,17 +113,15 @@ useful changes, it doesn't matter if the overall state is or is not stable.
 As a tenet of its design, Kubernetes uses lots of controllers that each manage
 a particular aspect of cluster state. Most commonly, a particular control loop
 (controller) uses one kind of resource as its desired state, and has a different
-kind of resource that it manages to make that desired state happen.
+kind of resource that it manages to make that desired state happen. For example, 
+a controller for Jobs tracks Job objects (to discover new work) and Pod object 
```

Comment:
I spotted a typo that's there from the original. Worth fixing at the same time?
```suggestion
a controller for Jobs tracks Job objects (to discover new work) and Pod objects
```

Reply:
Do the changes get squashed before being merged?